File: Mobile/ios/swift/Id.swift
//
//  Id.swift
//  SampleSwiftUI
//
//  Created by Karim Lalani on 1/21/20.
//  Copyright © 2020 Sertainty. All rights reserved.
//

import Foundation

public class Id {
    public class func publishToFile(_ callStatus: CallStatus, _ idFileSpec: String, _ doc: String, _ mods: Int) {
        uxpid_publishToFile(callStatus.handle, idFileSpec, doc, Int32(mods))
    }
}

File: Mobile/android/wrappers/java/org/Sertainty/Id.java
package org.Sertainty;

public class Id {

    public static void publishToFile(Sys.CallStatus callStatus, String idFileSpec, String doc, int mods)
    {
        publishToFile(callStatus.getHandle(), idFileSpec, doc, mods);
    }

    private native static void publishToFile(long callStatusHandle, String idFileSpec, String doc, int mods);
}
